performanceplatform/collector/main.py

```diff
@@ -99,6 +102,9 @@ def _log_collector_instead_of_running(entrypoint, args):
 
 def main():
     args = arguments.parse_args('Performance Platform Collector')
+    if 'collector_slug' in args and args.collector_slug:
```

Comment:
Neither is the other version because we are not assigning the args.get to anything

`'collector_slug' in args` is checking if the key is in the dictionary

`args.collector_slug` is checking that collector_slug is not [falsy](https://docs.python.org/2/library/stdtypes.html#truth-value-testing)

`args.get('collector_slug')` just does the checking if the value is in the dictionary and getting the value in one step, which is more succinct. In fact we don't even need to set a default value, so can just use `args.get('collector_slug')`, which will return `None` if `collector_slug` is not set, which will evaluate to false.